src/addrman.h

```diff
@@ -157,7 +157,7 @@ class CAddrInfo : public CAddress
 #define ADDRMAN_GETADDR_MAX_PCT 23
 
 //! the maximum number of nodes to return in a getaddr call
-#define ADDRMAN_GETADDR_MAX 2500
+#define ADDRMAN_GETADDR_MAX 1000
```

Comment:
I think commit message could be clearer, it doesn't remove the limit on AddrMan queries, rather bound it to the maximum size of ADDR message ?